### PR TITLE
Fixing docs. Add example and usage

### PR DESCRIPTION
**Pull request by pottekkat**
This is draft PR. I will finish it by EOD

### CODE REVIEW

**Comment by pottekkat** [comment]
@riibeirogabriel Please review and merge

**Review by riibeirogabriel — COMMENTED** [review]
Sounds good!
I don't have thinked about show first the usage example , it was good! About the package name, if we will change the name to learning-orchestra-client, we also need make this change in usage example code. The docs is better now!

**Comment by pottekkat** [comment]
> Sounds good!
> I don't have thinked about show first the usage example , it was good! About the package name, if we will change the name to learning-orchestra-client, we also need make this change in usage example code. The docs is better now!

I thought the name was learning-orchestra-client?

File: README.md [hunk 1]
```diff
@@ -1,259 +1,41 @@
...
-there are all class and each class methods, also have an example of workflow 
-using this package in a python code.
-
-## DatabaseApi
-
-### read_resume_files
-
-```python
-read_resume_files(pretty_response=True)
+pip install learning-orchestra-client
```
Comment (riibeirogabriel) [inline README.md:14]: In this line you did change the package name from learning_orchestra_client to learning-orchestra-client, I have thinked which you want change the package name, or you change accidentally?

**Review by riibeirogabriel — COMMENTED** [review]
@navendu-pottekkat See my comment in review.

File: README.md (hunk 1 above)
Comment (pottekkat) [reply README.md:14]: I checked pypi and it was pip install learning-orchestra-client

https://pypi.org/project/learning-orchestra-client/

Comment (riibeirogabriel) [reply README.md:14]: It is true! I have configured to use "learning_orchestra_client", but pypi seems changed the deploy to learning-orchestra-client, i tested the 2 ways and both works, but i don't know if installing whit a name way and using with another will work.

Comment (pottekkat) [reply README.md:14]: I think we will stick to "-" instead of using "_" because it is more conventional. So I will merge the PR.

**Pull request merged**